distribution/docs/src/main/resources/content/_architectures/migration-api.adoc

```diff
@@ -87,6 +88,9 @@ This will be the case when a migratable is later introduced in the software dist
 In order to create a `Migratable` for a module of the system, the `org.codice.ddf.migration.Migratable` interface must be implemented and the implementation must be registered under the `org.codice.ddf.migration.Migratable` interface as an OSGi service in the OSGi service registry.
 Creating an OSGi service allows for the migration API to lookup all implementations of `org.codice.ddf.migration.Migratable` and command them to export or import.
 
+==== `OptionalMigratable`
+This interface is designed as a tagged interfaces to identify optional migratables. An optional migratable will be skipped by default during the import phase. It can still be manually marked as mandatory when initiating the import phase.
```

Comment:
nit : `interfaces` -> `interfaces`

Reply:
You probably meant `interfaces` -> `interface` 😉 
Done